.github/workflows/bump-version-release.yaml

```diff
@@ -11,11 +11,12 @@ on:
       changelog:
         description: "Create changelog for release."
         required: false
-        default: false
+        default: true
         type: boolean
       changelog-config:
         description: "Changelog config path."
         required: false
+        default: "$GITHUB_ACTION_PATH/changelog-config.json"
```

Comment:
am not sure if that'd work like this. i think in order for this to work we need to add the config file in the `bump-version-release` action dir..

Reply:
you're right. after reading the [docs on GitHub variables](https://docs.github.com/en/actions/learn-github-actions/variables) I believe `$GITHUB_ACTION_PATH` actually doesn't work at all outside of a composite action... :/

Reply:
I've implemented a workaround